common/test/acceptance/tests/lms/test_oauth2.py

```diff
@@ -56,5 +56,6 @@ def test_accepting_redirects(self):
         self.oauth_page.confirm()
 
         # This redirects to an invalid URI.
-        query = self._qs(self.browser.current_url)
+        self.oauth_page.wait_for_element_absence('input[name=authorize]', 'Authorization button is not present')
+        query = self._qs(self.browser.current_url) or self._qs(self.browser.title)
         self.assertIn('code', query)
```

Comment:
I'm noticing now is that the intention of this particular test is to verify that a value for "code" was passed in **as a query parameter**. 

Could you please improve the test by changing what we are looking for from "code" to "code="? Thx.